phpcs.xml

```diff
@@ -65,4 +69,23 @@
   <exclude-pattern>docroot/profiles/custom/cgov_site/modules/custom/pdq_core/src/Commands/RevisionPrunerCommands.php</exclude-pattern>
   <exclude-pattern>docroot/profiles/custom/cgov_site/modules/custom/pdq_glossifier/js/plugins/cgov_glossifier/dialogs/glossify.js</exclude-pattern>
   <exclude-pattern>docroot/profiles/custom/cgov_site/modules/custom/pdq_glossifier/js/plugins/cgov_glossifier/dialogs/glossify.js</exclude-pattern>
+  <exclude-pattern>docroot/modules/custom/app_module/app_module.module</exclude-pattern>
```

Comment:
I take it these were all a bridge too far? Can you comment on the PR the issue number where these are captured to fix?

Reply:
One question, why are there 2 .install files here? Are just the update hooks the offenders?